spec/inject_method_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'inject_method'
+
+describe Array do
+  let(:array) { [1,2,3] }
+  
+  add = Proc.new {|plus, i|plus + i}
+  subtract = Proc.new {|plus, i|plus - i}
+  multiply = Proc.new {|plus, i|plus * i}
+  divide = Proc.new {|plus, i|plus / i}
```

Comment:
Space after closing `|` missing.<br>Space between { and | missing.<br>Space missing inside }.